Alignment-TUMAdsk-1/readme.md

```diff
@@ -0,0 +1,26 @@
+
+### Intent
+
+This scenario intents to provide the lowest-possible entry point to learn and understand the current IfcAlignment model. 
+It includes a single horizontal segment with its geometric and its semantic part. 
+
+The model was created using the GeometryGymIfc library (v 0.1.14) via nuget package manager.
+
+### Prerequisites
+
+None.
```

Comment:
I believe you still need to specify the default units, so a dependency on the Project Setup 1 would be necessary.